utilities.js

```diff
@@ -0,0 +1,26 @@
+function orderSingleObject(orderedKeys, object) {
```

Comment:
A utilities file isn't the best pattern as  its basically a shove random methods into a file cause we don't know where it should go file. Should put it in something more defined like `formatOutput.js` or `formatDisplay.js` or some other better name